frontend/src/core/l10n/actions.js

```diff
@@ -50,19 +51,39 @@ export function get(locales: Array<string>): Function {
     return async dispatch => {
         dispatch(request());
 
+        // Pseudo localization shows a weirdly translated UI, based on English.
+        // This is a development only tool that helps verifying that our UI
+        // is properly localized.
+        const usePseudoLocalization = (
+            process.env.NODE_ENV === 'development'
```

Comment:
We allow pseudo in release Firefox, any reason to restrict that to `development`?

Reply:
Why is it allowed in Firefox release? 

My reasoning was that I didn't want to run the risk of showing a broken UI to users on production, so I stayed on the safe side. I also don't see the value of doing this on stage or prod, but maybe I'm missing something?

Reply:
It's entertaining to just put someone's Firefox into pseudo-localized, just to show off that you can.

For Pontoon, it would also be something where you could audit the UI for non-localizable strings w/out having to do that on a local setup.

I'd definitely play around with that on prod ;-)

Reply:
You got me at "entertaining".